public/css/main.css

```diff
@@ -1,15 +1,15 @@
 ﻿/*icons*/
+
```

Comment:
Adding new characters to the various classes in my opinion results to an increase in the file size that is loaded to the frontend. Some of this css files may run to quite a number of classes. We could argue that added size isn't that huge but taking into consideration network latency issues and speed of loading pages, I think we should take every opportunity we get to make our pages faster. In this case I think we would be better off without the new line characters in the css file.

Reply:
Could we minify CSS and JS files before deployment too?